charts/dv-pod/templates/statefulset.yaml

```diff
@@ -348,8 +349,8 @@ spec:
         - name: cluster-lock
           projected:
             sources:
-            - secret:
-                name: {{ .Values.secrets.clusterlock }}
+            - configMap:
+                name: {{ .Values.configMaps.clusterlock }}
                 optional: true
```

Comment:
No need to mount the cluster lock to the validator container

Reply:
Also below here (again can't comment), we should be very careful about allowing persistence to be disabled for the VCs. If we're not careful we could cause a slashing. I think we should lean towards persistence being a must for them?